roles/sahara/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+sahara:
+  enabled: True
+  juno_url: http://tarballs.openstack.org/sahara/sahara-stable-juno.tar.gz
+  dashboard_url: http://tarballs.openstack.org/sahara-dashboard/sahara-dashboard-master.tar.gz
```

Comment:
I am not aware of how its fetching the source. I think its doing via openstack-source role but not entirely sure. Can you help in this please.